src/visualizers/SeqColor.ts

```diff
@@ -0,0 +1,390 @@
+import type p5 from 'p5'
+import {VisualizerDefault} from './VisualizerDefault'
+import type {VisualizerInterface} from '@/visualizers/VisualizerInterface'
+import {VisualizerExportModule} from '@/visualizers/VisualizerInterface'
+import type {SequenceInterface} from '../sequences/SequenceInterface'
+
+/** md
+
+# Primes and Sizes Visualizer
```

Comment:
The visualizer needs to be fully documented.

Reply:
mostly done now; still needs maybe another interesting example or two

Reply:
I'm quite happy with this now; ready for review

Reply:
Overall documentation is good.